pkgs/applications/emulators/86box/default.nix

```
@@ -43,7 +46,7 @@ stdenv.mkDerivation (finalAttrs: {
     pkg-config
     makeWrapper
     qt5.wrapQtAppsHook
-  ];
+  ] ++ lib.optional enableWayland extra-cmake-modules;
```

```suggestion
  ] ++ lib.optionals enableWayland [ extra-cmake-modules wayland-scanner ];
```

https://github.com/86Box/86Box/blob/c728f6cbf59735196e0ef69be514583c915e1d72/src/qt/CMakeLists.txt#L414

The reason it's working is that `strictDeps` isn't enabled

---

Thanks @Artturin ! I applied this and your other suggestion but I just wanted to understand a bit more about `strictDeps`. Should `strictDeps` be enabled here? It seems like maybe it should be whenever possible? If you have any information on it or even just a link to some documentation that would be greatly appreciated 😄 .

---

No docs but it's good to enable it when possible